omod/src/main/webapp/pages/results.gsp

```diff
@@ -143,7 +202,9 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
 <table id="list-results" cellspacing="0" cellpadding="2" width="100%" class="dataTable">
     <thead>
     <tr>
-        <th style="display: none"></th>
+        <th class="hidden"></th>
```

Comment:
@shivtej1505 I still do not understand the value of the first 3 columns which are hidden